## About
Have you ever wanted to have a `break` equivalent for outer function scopes?        
I think Kotlin does it, albeit in a different way.       
Here I demonstrate the simple use of `try..catch` and `throw` that allows you to exit with some value, bubbling upwards through function scopes.       
Don't you just *love exceptions*? No, only me?        
And to be clear `brake_` is a utility function, it is irrelevant to the method itself (throw-breaking).

